packages/sdk/src/FlagResolverClient.ts

```diff
@@ -134,13 +152,17 @@ export class FetchingFlagResolverClient implements FlagResolverClient {
         this.resolveTimeout,
         new ResolveError('TIMEOUT', 'Resolve timeout'),
       );
+      const start = new Date().getTime();
```

Comment:
Here it's a little nicer to do `Date.now()` which just gives a unix ms epoch. Date objects are generally avoided.